terra-jupyter-hail/Dockerfile

```diff
@@ -1,30 +1,18 @@
 FROM us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:0.0.1
 USER root
+ENV PIP_USER=false
 
-# Use Spark 2.2.0 which corresponds to Dataproc 1.2. See:
-#   https://cloud.google.com/dataproc/docs/concepts/versioning/dataproc-versions
-# Note: we are actually using Spark 2.2.1, but the Hail package is built using 2.2.0
-ENV SPARK_VER 2.4.0
-ENV SPARK_HOME=/usr/lib/spark
-
-# result of `gsutil cat gs://hail-common/builds/0.2/latest-hash/cloudtools-3-spark-2.2.0.txt` on 26 March 2019
-ENV HAILHASH daed180b84d8
-# ENV HAILJAR hail-0.2-$HAILHASH-Spark-$SPARK_VER.jar
-# ENV HAILPYTHON hail-0.2-$HAILHASH.zip
-# ENV HAIL_HOME /etc/hail
-ENV KERNELSPEC_HOME /usr/local/share/jupyter/kernels
-
-ENV PATH $SPARK_HOME:$SPARK_HOME/python:$SPARK_HOME/bin:$HAIL_HOME:$PATH
-ENV PYTHONPATH $PYTHONPATH:$HAIL_HOME/$HAILPYTHON:$HAIL_HOME/python:$SPARK_HOME/python:$JUPYTER_HOME/custom
+COPY scripts $JUPYTER_HOME/scripts
 
 # Note Spark and Hadoop are mounted from the outside Dataproc VM.
 # Make empty conf dirs for the update-alternatives commands.
-RUN apt install openjdk-8-jdk \
-&& pip3 install hail
-# mkdir -p /etc/spark/conf.dist && mkdir -p /etc/hadoop/conf.empty && mkdir -p /etc/hive/conf.dist \
```

Comment:
Hm.. do we not need to create empty `/etc/spark/conf` and `/etc/hadoop/conf` dirs anymore? Previously we did that so the actual dirs could be mounted into the container from outside.

Reply:
I am still confused about why we can remove this..

Reply:
![Screen Shot 2019-09-19 at 4 14 09 PM](https://user-images.githubusercontent.com/32771737/65277756-9a7ae280-daf8-11e9-99e2-0438f417b778.png)

I created a cluster with 3 workers...when I ran `hl.balding_nichols_model(3, 1000, 1000)._force_count_rows()`, all 3 workers have spikes similar to the screen shot.

Conversation with hail team
```
Qi Wang: I created a cluster with 3 workers, and all 3 workers have spikes when I run hl.balding_nichols_model(3, 1000, 1000)._force_count_rows()

Tim Poterba: ok, great!

Qi Wang: so I guess it's safe to say hail can talk to spark workers just fine?

Tim Poterba: yeah
```